packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -379,6 +379,45 @@ describe('postgresql', () => {
     expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
   })
+
+  test('introspection should load .env file with --print', async () => {
+    ctx.fixture('schema-only-postgresql')
+    expect.assertions(5)
+
+    try {
+      await DbPull.new().parse(['--print', '--schema=./prisma/using-dotenv.prisma'])
+    } catch (e) {
+      expect(e.code).toEqual('P1001')
+      expect(e.message).toContain(`fromdotenvdoesnotexist`)
+    }
+
+    expect(ctx.mocked['console.log'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.info'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+    expect(ctx.mocked['console.error'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
+  })
+
+  test('introspection should load .env file without --print', async () => {
+    ctx.fixture('schema-only-postgresql')
+    expect.assertions(5)
+
+    try {
+      await DbPull.new().parse(['--schema=./prisma/using-dotenv.prisma'])
+    } catch (e) {
+      expect(e.code).toEqual('P1001')
+      expect(e.message).toContain(`fromdotenvdoesnotexist`)
+    }
```

Comment:
Maybe `expect(...).toThrow()`/`expect(...).toThrowErrorMatchingSnapshot()`? Not sure if it will actually be better than `expect.assertions` + try/catch here, so feel free to ignore the suggestion if you think it won't.

Reply:
I had that before but here I wanted something that would not be updated like a snapshot. 
Main benefit is that we can't do a `--u` and update things without noticing, it's a stronger test I think, also if the message change around we won't need an update.

Downside is that `expect.assertions` is required, I think we could change that to `expect.hasAssertions()` in most cases.